Xamarin.Forms.Platform.UAP/ImageElementManager.cs

```diff
@@ -145,20 +145,27 @@ public static async Task UpdateSource(IImageVisualElementRenderer renderer)
 			}
 
 			var imageController = Element as IImageController;
+			
+			if (imageElement.LoadingSource != null)
+			{
+				var imageSource = await imageElement.LoadingSource.ToWindowsImageSourceAsync();
+				renderer.SetImage(imageSource);
```

Comment:
It looks like UWP is missing the checks that iOS and Android have to make sure the source hasn't changed :-/ 

Can you add those checks in ?